Author: Federico Ceratto <federico.ceratto@gmail.com>
Description: Prevents some of the empty __init__.py from being ignored by dh_pysupport

diff -Naurp turbogears.orig/i18n/data/__init__.py turbogears/i18n/data/__init__.py
--- turbogears.orig/i18n/data/__init__.py
+++ turbogears/turbogears/i18n/data/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/i18n/pygettext/__init__.py turbogears/i18n/pygettext/__init__.py
--- turbogears.orig/i18n/pygettext/__init__.py
+++ turbogears/turbogears/i18n/pygettext/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/identity/tests/__init__.py turbogears/identity/tests/__init__.py
--- turbogears.orig/identity/tests/__init__.py
+++ turbogears/turbogears/identity/tests/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/__init__.py turbogears/qstemplates/__init__.py
--- turbogears.orig/qstemplates/__init__.py
+++ turbogears/turbogears/qstemplates/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/qsbase/+package+/__init__.py turbogears/qstemplates/qsbase/+package+/__init__.py
--- turbogears.orig/qstemplates/qsbase/+package+/__init__.py
+++ turbogears/turbogears/qstemplates/qsbase/+package+/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/qsbase/+package+/templates/__init__.py turbogears/qstemplates/qsbase/+package+/templates/__init__.py
--- turbogears.orig/qstemplates/qsbase/+package+/templates/__init__.py
+++ turbogears/turbogears/qstemplates/qsbase/+package+/templates/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/quickstart/+package+/config/__init__.py turbogears/qstemplates/quickstart/+package+/config/__init__.py
--- turbogears.orig/qstemplates/quickstart/+package+/config/__init__.py
+++ turbogears/turbogears/qstemplates/quickstart/+package+/config/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/quickstart/+package+/tests/__init__.py turbogears/qstemplates/quickstart/+package+/tests/__init__.py
--- turbogears.orig/qstemplates/quickstart/+package+/tests/__init__.py
+++ turbogears/turbogears/qstemplates/quickstart/+package+/tests/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/qstemplates/quickstart/+package+/templates/__init__.py turbogears/qstemplates/quickstart/+package+/templates/__init__.py
--- turbogears.orig/qstemplates/quickstart/+package+/templates/__init__.py
+++ turbogears/turbogears/qstemplates/quickstart/+package+/templates/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/static/css/__init__.py turbogears/static/css/__init__.py
--- turbogears.orig/static/css/__init__.py
+++ turbogears/turbogears/static/css/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/static/images/__init__.py turbogears/static/images/__init__.py
--- turbogears.orig/static/images/__init__.py
+++ turbogears/turbogears/static/images/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/static/__init__.py turbogears/static/__init__.py
--- turbogears.orig/static/__init__.py
+++ turbogears/turbogears/static/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/static/js/__init__.py turbogears/static/js/__init__.py
--- turbogears.orig/static/js/__init__.py
+++ turbogears/turbogears/static/js/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/tests/catwalk_models/__init__.py turbogears/tests/catwalk_models/__init__.py
--- turbogears.orig/tests/catwalk_models/__init__.py
+++ turbogears/turbogears/tests/catwalk_models/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/tests/__init__.py turbogears/tests/__init__.py
--- turbogears.orig/tests/__init__.py
+++ turbogears/turbogears/tests/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/view/templates/__init__.py turbogears/view/templates/__init__.py
--- turbogears.orig/view/templates/__init__.py
+++ turbogears/turbogears/view/templates/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/widgets/templates/__init__.py turbogears/widgets/templates/__init__.py
--- turbogears.orig/widgets/templates/__init__.py
+++ turbogears/turbogears/widgets/templates/__init__.py
@@ -0,0 +1 @@ 
+# 


diff -Naurp turbogears.orig/widgets/tests/__init__.py turbogears/widgets/tests/__init__.py
--- turbogears.orig/widgets/tests/__init__.py
+++ turbogears/turbogears/widgets/tests/__init__.py
@@ -0,0 +1 @@ 
+# 

